entries/SWC-Proposed.md

```diff
@@ -0,0 +1,18 @@
+# Title
+Gas Siphon Attack
+
+## Relationships
+[CWE-470: Use of Externally-Controlled Input to Select Classes or Code](https://cwe.mitre.org/data/definitions/470.html)
+
+## Description
+When an account calls an address that belongs to a contract the contract's code will execute and the caller's account will pay the gas for this computation. Programs which send transactions, such as geth, estimate the gas needed automatically and will include the gas for that arbitrary computation. Therefore if the sender of a transaction auto generates the transaction without validation they may end up paying for gas used in unintended computation. Alone this is a form of griefing, but combined with a gas token structure it allows attackers to steal gas.
```

Comment:
I don't think this is a form of griefing; it doesn't prevent the intended computtaion from occurring.